draft-ietf-tls-ticketrequests.md

```diff
@@ -91,11 +91,12 @@ therefore bound the number of parallel connections they initiate by the number o
 in their possession, or risk ticket re-use.
 - Connection racing: Happy Eyeballs V2 {{?RFC8305}} describes techniques for performing connection
 racing. The Transport Services Architecture implementation from {{?TAPS=I-D.ietf-taps-impl}} also describes
-how connections can race across interfaces and address families. In cases where clients have early
-data to send and want to minimize or avoid ticket re-use, unique tickets for each unique
-connection attempt are useful. Moreover, as some servers may implement single-use tickets (and even
-session ticket encryption keys), distinct tickets will be needed to prevent premature ticket
-invalidation by racing.
+how connections can race across interfaces and address families. In such cases, clients may use
+more than one ticket while racing connection attempts in order to establish one successful connection.
+Requesting multiple tickets a priori equips clients with enough tickets to initiate connection racing while
+avoiding ticket re-use and ensuring that their cache of tickets does not empty during such races.
+Moreover, as some servers may implement single-use tickets (and even session ticket encryption keys),
```

Comment:
Do servers implement "session ticket encryption keys" or "single-use session ticket encryption keys"?

Reply:
Good point. I'll just drop the parenthetical to avoid confusion.